src/app/core/models/github-issue.model.ts

```diff
@@ -57,7 +91,7 @@ export class GithubLabel {
     if (this.isCategorical()) {
```

Comment:
Btw could we rename this function to `getLabelCategory()` and the one below to `getLabelValue()`. Now its slightly confusing haha cause both has the `getCategory..()` in it.

Reply:
Sure haha, I have renamed them to `getCategory` and `getValue` instead.

Reply:
Sure 👍